packages/e2e-tests/specs/performance/site-editor.test.js

```diff
@@ -117,7 +117,7 @@ describe( 'Site Editor Performance', () => {
 		await canvas().click(
 			'[data-type="core/post-content"] [data-type="core/paragraph"]'
 		);
-		await insertBlock( 'Paragraph' );
+		await insertBlock( 'Paragraph', { checkSelectedTab: true } );
```

Comment:
This needs to be revisited for this PR as it will probably affect the perf results. With this PR the results are contained to the selected tab, so since we have `patterns` as the first tab in site editor, we have to first click to the `blocks` tab before inserting the block.. --cc @youknowriad

Reply:
are you saying we need to add a step in the performance e2e tests so that the block inserter appears?

Reply:
yes